src/pudl/transform/eia861.py

```diff
@@ -811,61 +968,70 @@ def service_territory(tfr_dfs):
-            form (keys) correspond to normalized DataFrames of values from that page
-            (values).
+        raw_service_territory_eia861: Raw EIA-861 utility service territory dataframe.
 
     Returns:
-        dict: a dictionary of pandas.DataFrame objects in which pages from EIA861 form
-            (keys) correspond to normalized DataFrames of values from that page
-            (values).
+        The cleaned utility service territory dataframe.
     """
-    # No data tidying required
-    # There are a few NA values in the county column which get interpreted
-    # as floats, which messes up the parsing of counties by addfips.
-    type_compatible_df = tfr_dfs["service_territory_eia861"].astype(
-        {"county": "string"}
-    )
-    # Transform values:
-    # * Add state and county fips IDs
-    transformed_df = (
+    df = _pre_process(raw_service_territory_eia861)
+    # A little WV county sandwiched between OH & PA, got miscategorized a few times:
+    df.loc[(df.state == "OH") & (df.county == "Brooke"), "state"] = "WV"
+    df = (
+        # There are a few NA values in the county column which get interpreted
+        # as floats, which messes up the parsing of counties by addfips.
+        df.astype({"county": "string"})
         # Ensure that we have the canonical US Census county names:
-        pudl.helpers.clean_eia_counties(type_compatible_df, fixes=EIA_FIPS_COUNTY_FIXES)
+        .pipe(clean_eia_counties, fixes=EIA_FIPS_COUNTY_FIXES)
         # Add FIPS IDs based on county & state names:
-        .pipe(pudl.helpers.add_fips_ids)
+        .pipe(add_fips_ids)
+        .assign(short_form=lambda x: _make_yn_bool(x.short_form))
+        .pipe(_post_process)
+    )
+    # The Virgin Islands and Guam aren't covered by addfips but they have FIPS:
+    st_croix = (df.state == "VI") & (df.county.isin(["St. Croix", "Saint Croix"]))
+    df.loc[st_croix, "county_id_fips"] = "78010"
+    st_john = (df.state == "VI") & (df.county.isin(["St. John", "Saint John"]))
+    df.loc[st_john, "county_id_fips"] = "78020"
+    st_thomas = (df.state == "VI") & (df.county.isin(["St. Thomas", "Saint Thomas"]))
+    df.loc[st_thomas, "county_id_fips"] = "78030"
+    df.loc[df.state == "GU", "county_id_fips"] = "66010"
+
+    pk = (
+        Package.from_resource_ids()
+        .get_resource("service_territory_eia861")
+        .schema.primary_key
+    )
+    # We've fixed all we can fix! ~99.84% FIPS coverage.
+    df = (
+        df.dropna(subset=["county_id_fips"])
+        .sort_values(pk + ["county"])
+        .drop_duplicates(pk)  # Several hundred duplicates w/ different county names.
     )
-    transformed_df["short_form"] = _make_yn_bool(transformed_df.short_form)
-    tfr_dfs["service_territory_eia861"] = transformed_df
-    return tfr_dfs
+    return df
 
 
-def balancing_authority(tfr_dfs):
+@asset
+def clean_balancing_authority_eia861(
+    raw_balancing_authority_eia861: pd.DataFrame,
+) -> pd.DataFrame:
```

Comment:
Given that almost all the outputs of this module are at the `clean` step in their EtLT journey, I'm kind of inclined to name them with the `clean_` prefix when we write them to the DB.

This is the only table I did give that name to (and it's not written to the DB) because there's a subsequent transformation and cleanup that normalizes the `balancing_authority_eia861` table and really does finish it. The only other finished tables are `utility_assn_eia861`, `service_territory_eia861`, and `balancing_authority_assn_eia861 (all of which are association tables).

Curious what others think about the `raw_`, `clean_`, `norm_` => plain naming.

Reply:
`raw_`, `clean_`, and `norm_` make sense to me because I'm super familiar with how we structure our data. I'm worried people will be attracted to the `clean_` tables when they should really be looking at the `norm_` or output tables. If we're clear about the distinctions in our docs maybe this won't be a problem. 

Probably not for right now but down the line, we could adopt more conventional data modeling naming conventions. For example, our annual and entity tables roughly correspond with fact and dimension tables in the snowflake schema.

Reply:
I'd love to sit down and figure out what other people call these things and use those names so it's easier to communicate (and easier for folks who know about this stuff to contribute!)

I *think* that the tables we want people to use will be the ones that don't have any prefix at all... We should probably have 2 sets of documentation / data-dictionaries. One that's public-facing "analysis ready" denormalized data warehouse tables, and one that's all the normalized or skinny building blocks that go into constructing those tables, or earlier stages that are useful for understanding the journey the data took to get to you.

But maybe all of that doesn't live in the DB? Some of it is only in Dagster? I don't know what the right thing to do is.

Reply:
Hmm good questions. We could chat with Brent to get an additional opinion on naming conventions.

Reply:
Anyway I think it's Beyond The Scope of this PR. Just something to put a pin in.